### PR TITLE
Update sickchill package

### PR DESCRIPTION
**Pull request by miigotu**
Signed-off-by: miigotu <miigotu@gmail.com>

### CODE REVIEW

File: post_install.sh [hunk 1]
```diff
@@ -1,18 +1,23 @@
...
-# Copy the init script where it's useful
-cp /usr/local/sickchill/runscripts/init.freebsd /usr/local/etc/rc.d/sickchill
-chmod +x /usr/local/etc/rc.d/sickchill
+echo "Create sickchill user"
+pw user add sickchill -c "Sickchill" -d /nonexistent -s /usr/bin/nologin -w no
+chown -R sickchill:sickchill /usr/local/app/sickchill
+
+echo "Installing SickChill service"
+mkdir /usr/local/etc/rc.d
+cp /usr/local/app/sickchill/runscripts/init.freebsd /usr/local/etc/rc.d/sickchill
```
Comment (fireph) [inline post_install.sh]: I think this line needs to be:

`cp /usr/local/app/sickchill/contrib/runscripts/init.freebsd /usr/local/etc/rc.d/sickchill`

Comment (miigotu) [reply post_install.sh]: Can you make the PR?

**Comment by miigotu** [comment]
Actually, I see this is still open I will fix it later.

File: post_install.sh (hunk 1 above)
Comment (miigotu) [reply post_install.sh]: Fixed. Thanks for the review!

**Comment by miigotu** [comment]
@kmoore134 Can we get this side done as well?

**Review by kmoore134 — APPROVED** [review]

**Comment by kmoore134** [comment]
Yep, thanks for reminding me!

**Pull request merged**